__tests__/integration.test.js

```diff
@@ -151,3 +151,51 @@ describe("GET /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("POST 201: should add a comment to an existing article", () => {
+    const newComment = {
+      username: "butter_bridge",
+      body: "this is a test comment",
+    };
+
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        expect(response.body.author).toBe("butter_bridge");
+        expect(response.body.body).toBe("this is a test comment");
+        expect(response.body.article_id).toBe(1);
+        expect(response.body.votes).toBe(0);
+      });
+  });
+  test("POST 400: invalid article_id input", () => {
```

Comment:
This is more of a 'missing property' 400. Could you try implementing a missing property (ideally not `article_id`) 400 as well as an invalid `article_id` 400?